src/explore-education-statistics-frontend/src/services/tableBuilderService.ts

```diff
@@ -151,10 +137,7 @@ export default {
   ): Promise<PublicationSubjectMeta> {
     return dataApi.post('/meta/subject', query);
   },
-  getTableData(query: TableDataQuery): Promise<TableData> {
+  getTableData(query: TableDataQuery): Promise<FullTable> {
     return dataApi.post('/tablebuilder', query);
```

Comment:
Merge together in conjunction with
https://github.com/dfe-analytical-services/explore-education-statistics/pull/604